Added styles for active menu items

Quetan opened the pull request:
**Если все правильно понял, то сделал:**
1) Маленький скрипт для добавления классов .active, для пунктов меню, которые выбраны, а также изменение текста для кнопки с СМС (Вкл/Выкл)
2) Зеленый bg для активных пунктов меню
3) Красный bg при hover для уже активных пунктов, предполагающий исключение элемента
4) Кнопка, связанная с СМС меняет bg color, а также текст

**_Цепанул jQuery_** (в проекте вроде не нашел) , поскольку библиотека супер распространенная и значительно облегчила написание скрипта, но если будет необходимость можно переписать на чистый js

Fixes #3

Quetan committed:
Added styles for active menu items

SimePel commented:
Спасибо, за уделенное время. По-поводу PR. Во-первых, нельзя просто так взять и притащить библиотеку в чужой проект. Для этого должны быть какие-то очень веские причины. JQuery - это уже плохой инструмент и использовать его я бы не советовал вообще. 

С цветами, на мой взгляд, тоже все не гладко. Например,  когда выбираешь до этого неактивный элемент(прозрачный), он должен стать зеленым, но из-за hovera при удержании мышки он красный, только после выезжания курсора, элемент снова становится зеленым. Да и общий вид, мне как-то не зашел. Все-таки, наверное, если сделать только кастомные чекбоксы, чтобы тыкнул на любую часть элемента и менялось состояние с галочки на крестик, то будет поприятней.

Кстати, класс "menu-list", выглядит избыточным, можно было и без него обойтись.

А вот смс, можно и так оставить.

Quetan committed:
Refactored some styles for active menu items

Переписал все на чистый js, из-за особенностей которого пришлось кидать стили не на li, а на 'a' элемент. Сделал более легковесный вариант с цветами, убрал кринжёвые зеленые и красные цвета, режущие глаз (только на смс оставил, но сделал менее яркие), думаю такой вариант зайдет больше, но с цветами можно поиграться еще.
Убрал лишние классы с li элементов, убрал jQuery (хорошая библиотека, не надо так с ней :anguished: ). Без jQuery получилось немного более громоздко, но голову поломал как сделать на чистом js без косяков.

Quetan commented:
Поменял немного, убрал лишнего всякого... Чекбоксы делать это жуть, как по мне, сделал второй вариант, можно еще с цветами поиграться там конечно, но на твой вкус... jQuery убрал, переписал на чистый js.

SimePel commented:
Спасибо за продолжение. Я чувствую мы близки.

Во-первых, `onclick` нужно заменить на `element.addEventListener("click", (e) => {`. Пиши стрелочный синтаксис => вместо слова function. И, я решил везде двойные кавычки использовать.

Во-вторых, мне кажется зря сделал `toggle` как id, оставь как class. Из-за того, что сделал как id, ты мог не писать так: `.menu .menu-list #toggle`, а оставил бы только `#toggle`. (_хотя уже не актуально_)

В-третьих, к вот этому правилу есть вопросики `.menu .menu-list li a`:
- Зачем нужно `width: 100%;`? Если оно ничего не делает, то убери.
- Зачем тут `transition: background-color .3s ease-in-out;` background-color? У меня даже редактор его просто белым подсвечивает? Опять же, если без него работает, то можно убрать.

В-четвертых, ты заметил, что есть комментарий, который не соответствует действительности в my.css? Лучше их вообще все удали.

В-пятых, зеленый цвет для смсок, стал слишком тусклым, не находишь?

В-шестых, ты в каком редакторе пишешь? Почему код неотформатированный скинул? Я надеюсь ты как продвинутый на VScode сидишь?

Ну и на последок, я заметил твою пасхалку +_+
![image](https://user-images.githubusercontent.com/20608155/63403208-7e582880-c408-11e9-922a-8c8a06554464.png)

Quetan commented:
Все мелочи вылезают отчасти потому, что пишу всегда поздней ночью после своих штук на vue и уже перестаю их замечать) Ну оставшееся уже точно подправим..

Quetan committed:
Cleaned and refactored

Вроде все поменял, как хотел...

Quetan commented:
Подправил что ты хотел, если что еще не понравится кидай, поменяю... Сократил свой код, убрал лишнюю функцию и исправил пару мелочей..
Transition оставил, поскольку он работает и разница заметна, без него выглядит слишком резко... Остальные вещи действительно не нужны были, это я за собой не убрался с прошлых раз...
 Ну и idшник сменил на class, как и просил.

SimePel commented:
Спасибо, еще раз. Получилось замечательно. Как видишь не на много и сложнее с чистым js вышло. Как никак новый ecmascript выходит каждый год и js становится лучше.
